fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTable.java

```diff
@@ -38,7 +40,7 @@
     private Map<Column, Set<Interval>> workingIntervals;
     private final Object mutex = new Object();
     private final Map<String, String> logicalToPhysicalColumnNames;
-    private final Map<String, String> physicalToLogicalColumnNames;
+    private final Map<String, List<String>> physicalToLogicalColumnNames;
```

Comment:
Does `Set` fit better (since it should be a unique set of logical dimensions)? I'm not sure what are the advantages of just using `Collection`, fill me in if you have time, thanks!